environment/env.go

```diff
@@ -37,12 +37,15 @@ const (
 	// CassandraDefaultPort Cassandra default port
 	CassandraDefaultPort = "9042"
 
+	// which sql system to use
+	SQLDriver        = "SQL_DRIVER"
+	SqlDefaultDriver = "postgres"
 	// MySQLSeeds env
-	MySQLSeeds = "MYSQL_SEEDS"
+	SQLSeeds = "MYSQL_SEEDS"
```

Comment:
not sure whether this is a good idea or not:
using `MYSQL_SEEDS` env var for postgres is kind of confusing

how about introducing 2 new env vars `SQL_SEEDS` and `SQL_PORT` for SQL usage (both mysql and postgres)
if `SQL_SEEDS` is not set, also check `MYSQL_SEEDS` (same applies to `SQL_PORT` and `MYSQL_PORT`)

if `MYSQL_SEEDS` or `MYSQL_PORT` but not `SQL_SEEDS` and `SQL_PORT`, log a warn?

what do you think @samarabbas